lte/gateway/c/core/oai/oai_mme/CMakeLists.txt

```diff
@@ -31,9 +31,24 @@ endif ()
 
 add_executable(mme ${PROJECT_SOURCE_DIR}/oai_mme/oai_mme.c)
 
+# compile the needed macros
+
+create_proto_dir("orc8r" ORC8R_CPP_OUT_DIR)
+
+list(APPEND PROTO_SRCS "")
+list(APPEND PROTO_HDRS "")
+
+set(SMGR_ORC8R_CPP_PROTOS mconfig/mconfigs)
+generate_cpp_protos("${SMGR_ORC8R_CPP_PROTOS}" "${PROTO_SRCS}"
+  "${PROTO_HDRS}" ${ORC8R_PROTO_DIR} ${ORC8R_CPP_OUT_DIR})
+
+find_package(MAGMA_LOGGING REQUIRED)
```

Comment:
why is this needed?

Reply:
ok note for me: saw that the library is included as a dependency for `MConfigLoader` functions